### PR TITLE
finished with hometask 3

### PR DESCRIPTION
**Pull request by YanaLapitskaya**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,7 @@
 **/*
 !**/*.go
 !**/
-!.gitignore
\ No newline at end of file
+!.gitignore
+!Gopkg.lock
+!Gopkg.toml
+/vendor
```
Comment (wshaman) [inline .gitignore:7]: Thank you for keeping web clean. 
But I'm afraid, it was a part of a task to add vendor folder to repo.

File: task_3_1/main.go [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+type People []Person
+
+var dateLayout string = "2006-01-02"
+
+func (p People) Len() int {
+	return len(p)
+}
+
+func (p Person) String() string {
+	return fmt.Sprintf("%v, %v, %v \n", p.firstName, p.lastName, p.birthday.Format(dateLayout))
```
Comment (wshaman) [inline task_3_1/main.go:26]: Generally it's not advised to add "\n" to stringer func. E.g. one might want to add name inside of a phrase, like "Hello, %s, welcome to the web", and formatting will be broken.

File: task_3_1/main.go [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+}
+
+func (p People) Less(i, j int) bool {
+	if p[i].birthday.Equal(p[j].birthday) {
+		return strings.Compare(
```
Comment (wshaman) [inline task_3_1/main.go:31]: Awesome!

**Review by wshaman — APPROVED** [review]

File: task_3_1/main.go (hunk 3 above)
Comment (YanaLapitskaya) [reply task_3_1/main.go:31]: 😄

File: task_3_1/main.go (hunk 2 above)
Comment (YanaLapitskaya) [reply task_3_1/main.go:26]: other mentor (Ilya) also mentioned this. But what to use instead?

File: .gitignore (hunk 1 above)
Comment (YanaLapitskaya) [reply .gitignore:7]: i thought that `Gopkg.lock` and `Gopkg.toml` is enough to make sure that i actually run scripts...

**Review by VolhaVetrava — APPROVED** [review]

**Pull request merged**